configs/_base_/datasets/nuim-instance.py

```diff
@@ -40,8 +39,10 @@
             dict(type='Normalize', **img_norm_cfg),
```

Comment:
we need to remove `Normalize`, `Pad` and `ImageToTensor` in pipeline which has been done in `data_preprocessor` in 1.1 version